apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -2101,6 +2101,10 @@ export const platform: NavMenuConstant = {
             },
           ],
         },
+        {
```

Comment:
We should probably revisit the order of these at the end, I feel like "Your monthly invoice" and "How billing works" are placed a bit too low in comparison to other sections

Reply:
Yes 👍 All "old" pages except from the FAQs will be deleted though, I guess.